components/program_manager/spec/lib/time_client_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.describe TimeClient do
+  describe '.get_time' do
+    subject(:get_time) { described_class.time_at(timezone_name) }
+
+    context 'when timezone format is correct' do
+      let(:timezone_name) { 'America/Vancouver' }
+
+      before do
+        stub_request(:get, "http://worldtimeapi.org/api/timezone/America/Vancouver")
+          .to_return(
+            body: "{\"abbreviation\":\"PDT\",\"client_ip\":\"186.193.137.78\",\
```

Comment:
If I did this again I'd maybe put it as a normal hash and just have `.to_json` called on it for readability, and edibility - but just opinion